be/test/exprs/json_functions_test.cpp

```diff
@@ -182,6 +182,315 @@ TEST_F(JsonFunctionsTest, get_json_string_array) {
                         .ok());
 }
 
+TEST_F(JsonFunctionsTest, json_query_filter_simple) {
```

Comment:
please use the Parameterized Test to write full test cases for it, covering various expressions & various json structure. current test cases contain pretty much duplicate code